### PR TITLE
Added the holdout dataset and the python file

### PR DESCRIPTION
**Pull request by aryanVijaywargia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/HakinCodes/Malaria-Detection/blob/master/CONTRIBUTING.md)?

Yes
### Description

- Added a directory containing 200 images randomly sampled from both the infected and uninfected cells(100 per class)

- Inserted the python file in the scripts folder

- Pylint score - 9.55/10

### Checklist

- [ ✔️ ] I've read the contribution guidelines.
- [ ✔️ ] This PR mets all the requirements like Clean Code, proper README and Documentation

### CODE REVIEW

**Comment by VarthanV** [comment]
@aryanVijaywargia  pls do squash your commits into a single commit

**Comment by aryanVijaywargia** [comment]
> @aryanVijaywargia pls do squash your commits into a single commit

My bad!! I have rectified it.

File: Scripts/test_set.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import os
+import random
+import shutil  # importing the required libraries
+
+import cv2
+
+parasite_dir = r"C:\Users\navee\Hakin Codes\cell_images\Parasitized"
```
Comment (HarshCasper) [inline Scripts/test_set.py:7]: Remove references from your Local System

File: Scripts/test_set.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+uninfected_dir = r"C:\Users\navee\Hakin Codes\cell_images\Uninfected"
```
Comment (HarshCasper) [inline Scripts/test_set.py:8]: Remove references from your Local System

File: Scripts/test_set.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+def create_test_set(
+        parasite_dir=parasite_dir,
+        uninfected_dir=uninfected_dir,
+        samples=samples,
+        target_size=target_size,
+):
+
+    # creating the directory for the holdout dataset images
+    os.makedirs(r"..\holdout_dataset", exist_ok=True)
+    target_dir = r"C:\Users\navee\Hakin Codes\Malaria-Detection\holdout_dataset"
```
Comment (HarshCasper) [inline Scripts/test_set.py:23]: Remove references from your Local System

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Kindly run your Script through `PyLint` and add the PyLint Score in your PR to know the quality of the Code.

File: Scripts/test_set.py (hunk 1 above)
Comment (aryanVijaywargia) [reply Scripts/test_set.py:7]: Actually the kaggle dataset is present in my local system (from where I am importing the respective images) and not on my github branch (cause I dont think we need to upload the whole dataset on the repo). Thats why I am not sure as to how to remove it .

**Comment by aryanVijaywargia** [comment]
> Kindly run your Script through `PyLint` and add the PyLint Score in your PR to know the quality of the Code.

Done 👍 , kindly check the point 3 of description

**Pull request closed without merging**